### PR TITLE
[feat/fix] Most fewest goals scored

### PR DESCRIPTION
**Pull request by MiTOBrien**
spec_helper.rb
- Added require './lib/team_statistics.rb'

team_statistics_spec.rb
- Wrote tests for most_goals_scored and fewest_goals_scored - TESTS PASS

team_statistics.rb
- Wrote methods for most_goals_scored and fewest_goals_scored - TESTS PASS

season_statistics.rb
- My tests wouldn't run in rspec initially do to a "missing end" error in season_statistics.rb file.  I went in and found the missing end (It was actually an EXTRA end) and cleaned up the code some.  Better alignment, removed extra lines between methods, etc.  It will probably need to be cleaned up more later, but my goal was to be able to run my tests.

### CODE REVIEW

File: lib/team_statistics.rb [hunk 1]
```diff
@@ -13,9 +16,32 @@ def team_name(team_id)
...
+            if game.home_team_id == team_id && game.home_goals.to_i > goals
+                goals = game.home_goals.to_i
+            end
+        end
+        goals
+    end
+
+    def fewest_goals_scored(team_id)
+        goals = 99
+        @games.each do |game|
```
Comment (sethverrill) [inline lib/team_statistics.rb:36]: I've never seen this approach before, but it's cool

**Review by sethverrill — COMMENTED** [review]
Looks like there's some merge conflicts here. We should probably figure them out as a group

**Comment by BrodyNics77** [comment]
Looks good!

**Review by Jonathan-Atkins — COMMENTED** [review]
looks good - fixing merge conflict!

**Pull request merged**